system_tests/espresso-e2e/docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 version: '3.9'
 services:
   orchestrator:
-    image: ghcr.io/espressosystems/espresso-sequencer/orchestrator:main
```

Comment:
Wonder why we're using the `arbmusl` tag?

Reply:
Ah so this is the `musl` image from the `arb` branch? Maybe call the branch `arbitrum-integration` to be a bit more descriptive?

Reply:
Oh. I checkout a branch called `arb` in sequencer repo and run the `static-build` action manually. And it created this tag. I don't know if it is appropriate to do that. But I think it is convenient for us to track and update

Reply:
No problem. Let's update the branch name. Then the `tag` would be `arbitrum-integrationmusl`

Reply:
We usually use the non-static docker images (it shouldn't matter here though) but it's probably better to be consistent.

If you want the branch to have docker images published when new commits are pushed to it I think you need to add the branch to the github actions triggers. For example here: https://github.com/EspressoSystems/espresso-sequencer/blob/3f5beae8e9f121987ecdacebb87be7ca5c0ab99d/.github/workflows/build.yml#L7

Or you can call the branch `release-...` then it will match an existing branch trigger.

Reply:
I think a better way for the integration is to publish the images manually. That means the test won't be broken by any code update. And if some one is going to publish the new version with this tag, and he will be responsible to check all the tests.